src/dash.cpp

```diff
@@ -6,6 +6,32 @@
 
 int main(int argc, char *argv[])
 {
+     QApplication dash(argc, argv);
```

Comment:
looks like something got screwed up with the merge... might want to double check the diff here, looks like your version of the file was "stacked" ontop of the original

Reply:
Corrected the dash.cpp.  Not sure why the merge combined the files but good catch.